### PR TITLE
booPark/bugFixes to main

### PR DESCRIPTION
**Pull request by bitNibblesByte**
booPark/bugFixes

- fixed the bug where exclamation sprite disappears after the initial appearance
- fixed up exclamation sprite
- fixed up ENTER button sprite

### CODE REVIEW

File: sprites/spr_buttonkey/1dd8d5e0-59cf-471c-aeaf-e9e8e0bf7f68.png
Comment (jentel) [inline sprites/spr_buttonkey/1dd8d5e0-59cf-471c-aeaf-e9e8e0bf7f68.png:1]: Oof. Supposed to use the Resx text

File: objects/obj_prompt/Step_0.gml [hunk 1]
```diff
@@ -7,19 +7,32 @@ y += shift;
...
 		if (image_alpha <= 0) {
 			fadeMe = FADE_DONE;
-			alarm[0] = 2 // queue destroy
+			alarm[0] = 2; // queue destroy
 		}
 		break;
 	}
+	
+	case FADE_VISIBLE: {
+		 // The object is now fully visible, do any continuous behavior here
```
Comment (jentel) [inline objects/obj_prompt/Step_0.gml:30]: Have some weird formatting in here

File: objects/obj_prompt/Alarm_0.gml [hunk 2]
```diff
@@ -1,4 +1,9 @@
...
 
-// destroy me
-//instance_destroy();
\ No newline at end of file
+// If the fade-out is done, you can either deactivate the object or reset its properties
+// Example: Resetting the object to prepare it for re-entry (make it visible again)
+fadeMe = FADE_IN;  // Set back to fade-in state so it can reappear
+image_alpha = 0;    // Reset alpha to make it invisible
+shift = 1;          // Reset bobbing direction to default
+alarm[1] = 10;      // Reset bobbing alarm to continue the bobbing effect
```
Comment (jentel) [inline objects/obj_prompt/Alarm_0.gml:8]: shouldn't this be `alarm[0]`

File: objects/obj_prompt/Step_0.gml (hunk 1 above)
Comment (jentel) [reply objects/obj_prompt/Step_0.gml:30]: Also, these cases don't make any sense. Why is there nothing in them?

**Review by jentel — CHANGES_REQUESTED** [review]

File: objects/obj_prompt/Step_0.gml (hunk 1 above)
Comment (bitNibblesByte) [reply objects/obj_prompt/Step_0.gml:30]: Does it need to exist in order for what's in Alarm 0 to take affect?

File: objects/obj_prompt/Alarm_0.gml (hunk 2 above)
Comment (bitNibblesByte) [reply objects/obj_prompt/Alarm_0.gml:8]: but bobbing alarm is in alarm[1], so should i be moving the bobbing effect to alarm[0] then?

Comment (jentel) [reply objects/obj_prompt/Alarm_0.gml:8]: lol code acts weird if taken out so it must remain

**Review by jentel — APPROVED** [review]

**Pull request merged**